Signal/src/Jobs/SessionResetJob.swift

```diff
@@ -30,9 +30,9 @@ class SessionResetJob: NSObject {
             self.storageManager.deleteAllSessions(forContact: self.recipientId)
 
             DispatchQueue.main.async {
-                let endSessionMessage = EndSessionMessage(timestamp:NSDate.ows_millisecondTimeStamp(), in: self.thread)
+                let endSessionMessage = EndSessionMessage(timestamp: NSDate.ows_millisecondTimeStamp(), in: self.thread)
 
-                self.messageSender.send(endSessionMessage, success: {
+                self.messageSender.enqueue(endSessionMessage, success: {
                     Logger.info("\(self.TAG) successfully sent EndSession<essage.")
                     let message = TSInfoMessage(timestamp: NSDate.ows_millisecondTimeStamp(),
```

Comment:
`OWSMessageSender` confusingly used the same naming pattern `sendMessage:` for two different sets of methods: 

* Methods for enqueuing outgoing messages.
* Methods for trying to deliver messages to the service.

I've renamed these methods in an attempt to make the code more readable.